fastly/errors.go

```diff
@@ -149,6 +149,18 @@ var ErrMissingCertBlob = errors.New("missing required field 'CertBlob'")
 // a "IntermediatesBlob" key, but one was not set.
 var ErrMissingIntermediatesBlob = errors.New("missing required field 'IntermediatesBlob'")
 
+// ErrMissingTLSCertificate is an error that is returned from an input struct that requires
+// a "TLSCertificate" struct, but one was not set.
+var ErrMissingTLSCertificate = errors.New("missing required field 'TLSCertificate'")
+
+// ErrMissingTLSConfiguration is an error that is returned from an input struct that requires
+// a "TLSConfiguration" struct, but one was not set.
+var ErrMissingTLSConfiguration = errors.New("missing required field 'TLSConfiguration'")
+
+// ErrMissingTLSDomainID is an error that is returned from an input struct that requires
```

Comment:
Looks like this should have been updated when the variable name was updated...

```suggestion
// ErrMissingTLSDomain is an error that is returned from an input struct that requires
```